spec/schema.json

```diff
@@ -361,10 +361,46 @@
         }
       }
     },
+    "/transactions/{transaction_id}/limited-partnership/general-partners": {
```

Comment:
not sure if this should be partners (plural). I was thinking it would be better to have /general-partner as it is the same resource we use for post/put/patch etc..

Reply:
same applies for limited-partner(s)

Reply:
didn't see the teams chat, i will ask on there

Reply:
it consistent with other CH apis, so it's ok